scipy/stats/_continuous_distns.py

```diff
@@ -970,6 +975,73 @@ def _entropy(self):
 expon = expon_gen(a=0.0, name='expon')
 
 
+## Exponentially Modified Gaussian (exponential distribution
+##  convolved with a Gaussian)
+class expongauss_gen(rv_continuous):
+    """An exponentially modified Gaussian continuous random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `expongauss` is::
+
+        expongauss.pdf(x, K) = 1/(2*K) exp(1/(2 * K**2)) exp(-x / K) * erfc(- (x - 1/K) / sqrt(2))
+           
+    where the shape parameter ``K`` > 0.
+
+    It can be thought of as the sum of a normally distributed random
+    value with mean ``loc`` and sigma ``scale`` and an exponentially
+    distributed random number with a pdf proportional to exp(-lambda * x)
+    where lambda = (``K`` * ``scale``)**(-1).
+
+    .. versionadded:: 0.16.0
+
+    %(example)s
+
+    """
+    def _argcheck(self, K):
+        return (K > 0)
+    
+    def _rvs(self, K, loc=0, scale=1):
+        invlam = scale * K
+        expval = self._random_state.standard_exponential(self._size) * invlam
+        gval = loc + scale * self._random_state.standard_normal(self._size)
+        return expval + gval
```

Comment:
I don't think this does what you want it to. loc and scale are not forwarded from `rvs` to `_rvs`. In `rv_continuous.rvs`, https://github.com/scipy/scipy/blob/master/scipy/stats/_distn_infrastructure.py#L869 peels off `loc, scale` from the `*args` and `**kwds`, and then https://github.com/scipy/scipy/blob/master/scipy/stats/_distn_infrastructure.py#L891rescales the output from `_rvs`.